### DISCUSSION TITLE
My Account Suspended With No Warning or Notice

### DISCUSSION THREAD

**Discussion by KaysGramma** [mastodon/mastodon#24058] [Q&A]
Can anyone help me, point me in the right direction -- I logged on to mstdn.social this evening to discover my account had been suspended 2 days ago. I wasn't notified, never got an email. The "past strikes" is empty -- the only strike is my suspension. I tried sending an email to security@mastodon, a private message on Twitter & filed an appeal (although I have no idea what I'm appealing since I have no clue whats behind this).  I don't get whats happening and this makes no sense. Even if I DID do something wrong -- why wouldn't they notify me?

**Comment by trwnh** [comment]
this is the discussion forum for the mastodon project, which has nothing to do with mstdn.social. you may wish to contact the admins of mstdn.social instead (`hello@mstdn.social`)
